proposals/lip-0050.md

```diff
@@ -190,21 +291,17 @@ This module does not expose any functions.
 
 This function provides the legacy address and balance of the corresponding legacy accounts.
 
-##### Parameters
-
-* inputParameter1: `publicKey`, an Ed25519 public key
-
 ##### Returns
 
 Either an object with the properties `legacyAddress` and `balance`, where `legacyAddress` is the legacy address for `publicKey` and `balance` is the balance of the corresponding legacy account, or `undefined`.
 
 ##### Execution
 
 ```python
-getLegacyAccount(publicKey)
-    let legacyAddress be the legacy address of publicKey
+def getLegacyAccount(publicKey: PublicKeyEd25519) -> dict:
+    legacyAddress = getLegacyAddress(publicKey)
     if there exists no entry in the legacy accounts substore with store key equal to legacyAddress:
-        return undefined
+        return {}
```

Comment:
for the endpoint, maybe it's better to return empty value like
```
{
  legacyAddress,
  balance: '0'
}
```

Reply:
done